GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -88,6 +91,34 @@ public IReadOnlyList<FileStatusWithDescription> Reload()
                     }
                 }
 
+                // Show diff with base regarding default branch of default remote
+                IReadOnlyList<string> remotes = module.GetRemoteNames();
+                if (remotes.Count == 0)
+                {
+                    return fileStatusDescs;
+                }
+
+                string defaultRemote = remotes.FirstOrDefault(r => r is "origin" or "upstream") ?? remotes[0];
```

Comment:
This should be configurable, would be nice to see origin/upstream first in the remote lists 
Mine are not called that though.
(Similar discussed for prioritized branches.)

To see the tracked remote branch would be useful though.

Reply:
Yes, these heuristics do not work well enough. Even per-repo settings will not suffice, f.i. if working on a release branch.

f48ca3f9905fc723d77c9c00919075587d9585f3 seems to be nearer to what I want - in combination with multi-selection. Tracked remote branch and remote HEAD are further candidates for such multi-selections.